metrix/src/parser.rs

```diff
@@ -29,12 +29,14 @@ pub struct BaseExpression {
     value: Value,
 }
 
+#[derive(Debug)]
 pub enum FieldType {
     RootField(Field),
     NestedField(Field),
     TerminalField(()),
 }
 
+#[derive(Debug)]
```

Comment:
I'm not 100% sure we want to leave these on our structs

Reply:
we can build release with config: https://users.rust-lang.org/t/will-derive-debug-be-compiled-in-release-build/30388
#[cfg_attr(debug_assertions, derive(Debug))]